plenty_api/api.py

```diff
@@ -705,6 +705,66 @@ def plenty_api_get_variations(self,
             query=query,
             lang=lang)
 
+    def plenty_api_get_stock(self,
+                             refine: dict = None,
+                             lang: str = ''):
+        return self.plenty_api_generic_get(
+            domain='stockmanagement',
+            refine=refine,
+            lang=lang)
+
+    def plenty_api_get_storagelocations(self,
+                                        warehouse_id: int,
+                                        refine: dict = None,
+                                        additional: list = None,
+                                        lang: str = ''):
+        return self.plenty_api_generic_get(
+            domain='warehouses',
+            path=f'/{warehouse_id}/stock/storageLocations',
+            refine=refine,
+            additional=additional,
+            lang=lang)
+
+    def plenty_api_get_variation_stock_batches(self, variation_id):
+        # get all warehouses for this item
+        refine = {'variationId': variation_id}
+
+        # returns only locations with positive stock
+        stock = self.plenty_api_get_stock(refine=refine)
+        warehouses = [s['warehouseId'] for s in stock]
+
+        # get storage data from everywhere
+        # todo – think about romoving non-essenital keys
```

Comment:
To be honest I’m not sure what I originally meant here anymore. I think it was about removing keys from the returned data, because `plenty_api_get_storagelocations` returns more data than necessary here.

Reply:
Okay, then I would say we drop the comment.